app/views/home/index.html.haml

```diff
@@ -132,6 +132,8 @@
       %a.host{ href: "https://coderit.org", target: "_blank" }
         = image_tag 'coderit_logo.png'
 
+#logo-footer
+
```

Comment:
Sorry, one more:

<img width="1071" alt="screenshot 2017-12-23 14 48 31" src="https://user-images.githubusercontent.com/1441807/34322310-bc9b7ce0-e7f1-11e7-8517-5bc82056bed2.png">

I think this would look better if the margins above/below the graphic were much smaller, similar to the mocks where it's nearly the same space between [footer end] -> [page end]

<img width="1421" alt="screenshot 2017-12-23 14 56 23" src="https://user-images.githubusercontent.com/1441807/34322312-c0650bca-e7f1-11e7-9a2d-77d3ff7e528a.png">